src/style.css

```diff
@@ -1 +1,69 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc);
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.input-wrapper__big input {
```

Comment:
use class selectors instead of tag, and it's redundant to use nested selectors here

Reply:
Also add `box-sizing: border-box;` to fix issue with width of input